src/configurations/destinations/fb_custom_audience/db-config.json

```diff
@@ -11,13 +11,12 @@
     "saveDestinationResponse": true,
     "includeKeys": ["oneTrustCookieCategories", "consentManagement"],
     "excludeKeys": [],
```

Comment:
add back shopify - we came across an existing connection while reviewing prod fb custom audience sources yesterday with active data flowing in